### ISSUE TITLE
Rename the project towards gRPC?

### ISSUE THREAD

**Issue by iomz** [Interactions-HSG/xarm-commander#35]
From the discussion in #32 perhaps it makes more sense (and even attractive) to rename this project to "xarm-grpc" or something such?

**Comment by jo-bru** [comment]
Yes, indeed. This repo will hold then the `xarm-commander` as a testing and evaluation interface and the `xarm-deamon` (maybe something like `xarm-handler` would fit better as it's not only for the connection handling anymore, right?) as the gRPC server?
